docs/samples/custom/torchserve/docs/canary.md

```diff
@@ -0,0 +1,110 @@
+# Canary Rollouts
+
+## Deployment yaml
+
+```yaml
+apiVersion: "serving.kubeflow.org/v1beta1"
+kind: "InferenceService"
+metadata:
+ name: "torchserve"
```

Comment:
on yaml example it all uses name `torchserve-custom`, make sure they are consistent.